Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2063,6 +2063,9 @@ public async Task PreloadSparseStateWithTrustedPeers()
                     }),
                     trustedStateValidators: trustedPeers);
 
+                // Await 1 second to make sure all progresses is reported.
+                await Task.Delay(1000);
```

Comment:
How about using [`EventWaitHandle`][1] to wait until all progresses are reported?

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.threading.eventwaithandle?view=netstandard-2.0